python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -528,24 +543,26 @@ class RandomForestClassifier(Base):
 
         return ctypes.c_void_p(mod_handle).value
 
-    def _concatenate_treelite_handle(self, treelite_handle):
+    def _concatenate_treelite_handles(self, treelite_handles):
         cdef ModelHandle concat_model_handle = NULL
         cdef vector[ModelHandle] *model_handles \
             = new vector[ModelHandle]()
         cdef uintptr_t mod_ptr
-        for i in treelite_handle:
+        for i in treelite_handles:
             mod_ptr = <uintptr_t>i
             model_handles.push_back((
                 <ModelHandle> mod_ptr))
 
+        # TODO: free the old treelite model here
         concat_model_handle = concatenate_trees(deref(model_handles))
         cdef uintptr_t concat_model_ptr = <uintptr_t> concat_model_handle
         self.treelite_handle = concat_model_ptr
-        cdef vector[unsigned char] pbuf_mod_info = \
```

Comment:
question, if we are not clearing the old `self.model_pbuf_bytes` then when the code runs the `_get_protobuf_bytes` function will it not just use the old `self.model_bytes` instead of using the new `self.treelite_handle`  to obtain the bytes for the concatenated model